pages/course/[course]/assignment/[assignment]/index.tsx

```diff
@@ -1,8 +1,8 @@
 import { GetServerSidePropsContext, NextPage } from "next";
-import Sidebar from "../../../../Components/Sidebar";
+import Sidebar from "../../../../../Components/Sidebar";
 import Badge, { BadgeVariant } from "Components/Badge";
-import withProtected from "../../../../util/withProtected";
-import { queryParamToNumber } from "../../../../util/misc";
+import withProtected from "../../../../../util/withProtected";
```

Comment:
misc: we should alias the app root with @ or ~ or something to prevent long relative chains. no change needed for this PR

Reply:
Agreed was going to bring it up after sprint meeting, noticed it while working